### PR TITLE
resource/alicloud_sae_application: Added the field php, image_pull_secrets, programming_language, command_args_v2, custom_host_alias_v2, oss_mount_descs_v2, config_map_mount_desc_v2, liveness_v2, readiness_v2, post_start_v2, pre_stop_v2, tomcat_config_v2, update_strategy_v2, nas_configs, kafka_configs, pvtz_discovery_svc

### PR DESCRIPTION
**Pull request by MrWolong**

### CODE REVIEW

**Review by xiaozhu36 — APPROVED** [review]

**Review by xiaozhu36 — APPROVED** [review]

**Review by xiaozhu36 — APPROVED** [review]

**Review by xiaozhu36 — APPROVED** [review]

File: alicloud/resource_alicloud_sae_application_test.go [hunk 1]
```diff
@@ -139,19 +140,19 @@ func testSweepSaeApplication(region string) error {
 }
 
 // package_type = Image
-func TestAccAlicloudSAEApplication_basic0(t *testing.T) {
+func TestAccAliCloudSAEApplication_basicImage(t *testing.T) {
```
Comment (MrWolong) [inline alicloud/resource_alicloud_sae_application_test.go:143]: image

File: alicloud/resource_alicloud_sae_application_test.go [hunk 2]
```diff
@@ -1069,136 +1112,1425 @@ func TestAccAlicloudSAEApplication_basic2(t *testing.T) {
...
-				ImportStateVerifyIgnore: []string{"auto_enable_application_scaling_rule", "batch_wait_time", "config_map_mount_desc"},
+				ImportStateVerifyIgnore: []string{"auto_config", "auto_enable_application_scaling_rule", "change_order_desc", "deploy"},
 			},
 		},
 	})
 }
 
-func TestAccAlicloudSAEApplication_basicTags(t *testing.T) {
+// package_type = Php
+func TestAccAliCloudSAEApplication_basicPhp(t *testing.T) {
```
Comment (MrWolong) [inline alicloud/resource_alicloud_sae_application_test.go:1176]: 本地测试已pass
1433e6a193c870cb01f298a979265cde

File: alicloud/resource_alicloud_sae_application_test.go [hunk 3]
```diff
@@ -792,62 +818,88 @@ func TestAccAlicloudSAEApplication_basic1(t *testing.T) {
...
-				ImportStateVerifyIgnore: []string{"auto_enable_application_scaling_rule", "batch_wait_time", "config_map_mount_desc"},
+				ImportStateVerifyIgnore: []string{"auto_config", "auto_enable_application_scaling_rule", "change_order_desc", "deploy"},
 			},
 		},
 	})
 }
 
 // package_type = War
-func TestAccAlicloudSAEApplication_basic2(t *testing.T) {
+func TestAccAliCloudSAEApplication_basicWar(t *testing.T) {
```
Comment (MrWolong) [inline alicloud/resource_alicloud_sae_application_test.go:828]: 本地测试已pass
dfa3697c43907064d2f19693356d712b

File: alicloud/resource_alicloud_sae_application_test.go [hunk 4]
```diff
@@ -1069,136 +1112,1425 @@ func TestAccAlicloudSAEApplication_basic2(t *testing.T) {
...
+				ResourceName:            resourceId,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"auto_config", "auto_enable_application_scaling_rule", "change_order_desc", "deploy"},
+			},
+		},
+	})
+}
+
+func TestAccAliCloudSAEApplication_basicTags(t *testing.T) {
```
Comment (MrWolong) [inline alicloud/resource_alicloud_sae_application_test.go:2487]: 本地测试已pass
094548023c0cfabf2affe457c3a46f64

**Review by xiaozhu36 — APPROVED** [review]

**Review by xiaozhu36 — APPROVED** [review]

**Pull request merged**